amxmodx/modules.cpp

```diff
@@ -215,8 +215,13 @@ int load_amxscript_internal(AMX *amx, void **program, const char *filename, char
 
 			amx->flags |= AMX_FLAG_DEBUG;
 		} else {
-			ke::SafeStrcpy(error, maxLength, "Plugin not compiled with debug option");
-			return (amx->error = AMX_ERR_INIT);
+			const char * pluginname = filename;
+			// Easy extract pluginname from file path.
+			while(strstr(pluginname,"/")) pluginname = strstr(pluginname,"/") + 1;
+			while(strstr(pluginname,"\\")) pluginname = strstr(pluginname,"\\") + 1;
+			// Just skip debug info if not found.
+			AMXXLOG_Log("[AMXX] Plugin not compiled with debug option. Ignore debug. ( plugin \"%s\" )", pluginname);
```

Comment:
Try to follow local C++ style, "*" near name and spaces between keywords/tokens.

			const char *pluginname = filename;
			// Easy extract pluginname from file path.
			while (strstr(pluginname, "/"))
				pluginname = strstr(pluginname, "/") + 1;
			while (strstr(pluginname, "\\"))
				pluginname = strstr(pluginname, "\\") + 1;

And remove extra spaces in logging:

			AMXXLOG_Log("[AMXX] Plugin not compiled with debug option. Not debugging plugin \"%s\"", pluginname);

Reply:
Thanks, fixed in https://github.com/alliedmodders/amxmodx/pull/1027/commits/2b4111ebfd7a06edbec7437d0f2e90560870605f

Reply:
@dvander also added `#ifdef ` check in https://github.com/alliedmodders/amxmodx/pull/1027/commits/d4f896960179c3f990873a030f40b7f6babb7c38